src/card.tsx

```diff
@@ -1070,12 +1235,13 @@ export const PosterCard = React.forwardRef<HTMLDivElement, PosterCardProps>(
         const {textPresets} = useTheme();
 
         const isTouchable = touchableProps.href || touchableProps.to || touchableProps.onPress;
-        const overlayStyle = styles.touchableCardOverlayMedia;
 
         return (
             <CardContainer
                 width={width}
                 height={height}
+                dataAttributes={{...dataAttributes, 'component-name': 'PosterCard'}}
```

Comment:
Maybe you can use `getPrefixedDataAttributes` here

Reply:
no, that's already done inside CardContainer